config/query.config.prod.js

```diff
@@ -0,0 +1,8 @@
+const db = require('../config/db.config.prod');
+
+const query = (sqlQuery, cb) => {
+    // filter out reserver by default because we want tickets that have not been taken yet
```

Comment:
Surely this comment should go with whatever is calling this method (providing the SQL)?